scripts/README.md

```diff
@@ -71,3 +71,41 @@ EVC_IDS='d33539656d8b40,095e1d6f43c745' priority python3 scripts/002_unset_spf_a
 ```
 
 - After that, `kytosd` should be restarted just so `mef_eline` EVCs can get fully reloaded in memory with the expected primary and secondary constraints, this would be the safest route.
+
+### Change ``tag_type`` from integer to string type
```

Comment:
@Alopalao don't forget to map a issue as a reminder on kytos-ng.github.io repo similar to this one https://github.com/kytos-ng/kytos-ng.github.io/issues/60 to mention this script in the release notes. When reading the changlog that'd be obvious but let's facilitate for us and have an issue for this too, thanks.

Reply:
Created, [issue](https://github.com/kytos-ng/kytos-ng.github.io/issues/62)